### DISCUSSION TITLE
Changing rotation points in GLB Files

### DISCUSSION THREAD

**Discussion by HM3D** [cables-gl/cables_docs#331] [Answers & Questions]
Hi everyone!

I just wanted to ask what is the best workflow for changing the pivot points for different meshes inside a GLB file.

For example: suppose I have a GLB file of a robotic arm and I want to rotate each joint with a slider, how could this be accomplished?

Thank you,

HM

**Comment by steam0r** [comment]
i had some success with this approach, translate, rotate, translate back (lower translates 300 on x, upper -300 on x):

![Bildschirmfoto 2022-08-29 um 18 13 50](https://user-images.githubusercontent.com/151271/187246503-aa142123-bb02-4500-a05d-2a3ac0db4a81.png)

not sure if this can be easily applied to bones and such though...

**Reply by HM3D** [reply]
Ah! Cool -- I'll give that a try, thank you :D

At present I am separating my GLB file into separate GLB files, one per joint, and then parenting the transformation to an invisible shape made in Cables...very long and contrived....
